solana-programs/anchor/audius-data/tests/track-social-actions.ts

```diff
@@ -186,6 +213,47 @@ describe("track-actions", function () {
     );
   });
 
+  it("Delegate reposts a track", async function () {
+    const delegate = await testCreateUserDelegate({
+      adminKeypair,
+      adminStorageKeypair: adminStorageKeypair,
+      program,
+      provider,
+      ...getURSMParams(),
+    });
+
+    const tx = await addTrackRepost({
```

Comment:
```suggestion
    const txHash = await addTrackRepost({
```
? since below we say `info = getTransaction`